### PR TITLE
feat: eventcard component

### PR DESCRIPTION
**Pull request by MitaliShah**
## Description

Create a reusable Event Card Component

## Type

- [ ] Bug Fix
- [x] New Feature
- [ ] Code Refactor
- [ ] Documentation
- [ ] Other

## Related Issue
[Landing: Events: Event Card #6](https://github.com/Women-Coding-Community/wcc-frontend/issues/6)

## Screenshots
[Mobile](image

[Desktop](image

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] I checked and followed the [contributor guide](../CONTRIBUTING.md)
- [x] I have tested my changes locally.
- [x] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: src/components/index.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export { NavBar } from './NavBar';
+export { default as EventCard } from './EventCard';
```
Comment (stepsen89) [inline src/components/index.tsx:2]: can we align all exports to be the same as the others please? so you don't need export default in the EventCard itself

File: src/pages/index.tsx [hunk 2]
```diff
@@ -25,6 +25,7 @@ const HomePage = ({ data, error }: HomePageProps) => {
 
   return (
     <div>
+      <EventCard data={data.eventsSection} />
```
Comment (stepsen89) [inline src/pages/index.tsx:28]: this won't be used here, but inside the events component (the one which displays two/or one on mobile) - you can leave it there for now if you want to work on the events component overall too 👍

File: src/components/EventCard.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+import { Event } from '../utils/types';
+
+type EventCardProps = {
+  data: {
+    events: Event[];
+  };
+};
+
+const EventCard = ({ data }: EventCardProps) => {
+    const event = data.events[0];
```
Comment (stepsen89) [inline src/components/EventCard.tsx:15]: you can pass it only one event, this card will be used to map over the whole events array, so just a single one. but you can update it later too 👍

File: src/pages/index.tsx (hunk 2 above)
Comment (MitaliShah) [reply src/pages/index.tsx:28]: Yes, I'm gonna work on events component.

File: src/components/EventCard.tsx (hunk 3 above)
Comment (MitaliShah) [reply src/components/EventCard.tsx]: I'll update it when I work on events component.

File: src/components/index.tsx (hunk 1 above)
Comment (MitaliShah) [reply src/components/index.tsx]: Updated the export as the others.

File: public/home.json [hunk 4]
```diff
@@ -79,7 +79,7 @@
         "description": "Join us for a discussion of this essential guide to writing clear and maintainable code!\n",
         "images": [
           {
-            "path": "https://cloudprovider.com/image.png",
+            "path": "/event-mobile-img.png",
```
Comment (dricazenck) [inline public/home.json:82]: Considering we are going to use later the same structure, what about the create a folder events and move the image there?

Comment (MitaliShah) [reply public/home.json:82]: Hey @dricazenck 
I'm going to work on the overall event component that holds all the even cards. Can I update it when I do? Let me know.

**Review by stepsen89 — APPROVED** [review]

File: public/home.json (hunk 4 above)
Comment (dricazenck) [reply public/home.json:82]: all good :)

**Pull request merged**